src/vignette/storage/core.clj

```diff
@@ -1,7 +1,8 @@
 (ns vignette.storage.core
   (:require [vignette.media-types :as mt]
             [vignette.storage.protocols :refer :all]
-            [vignette.util.query-options :as q]))
+            [vignette.util.query-options :as q]
+            [vignette.storage.static-assets :as sa]))
```

Comment:
This namespace shouldn't need to know anything about a specific storage implementation.